metadata-db-app/src/cmr/metadata_db/config.clj

```diff
@@ -6,6 +6,11 @@
    [cmr.oracle.connection :as conn]
```

Comment:
It seems like the code shouldn't really be here. Since the tables are in the ingest DB, this should go with the ingest jobs.